Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -418,7 +423,7 @@ void doEsirkepovDepositionShapeN (const amrex::ParticleReal * const xp,
                     for (int imode=1 ; imode < n_rz_azimuthal_modes ; imode++) {
                         // The factor 2 comes from the normalization of the modes
                         // The minus sign comes from the different convention with respect to Davidson et al.
-                        const Complex djt_cmplx = -amrex::Real(2.)*I*(i_new-1 + i + xmin*dxi)*wq*invdtdx/(amrex::Real)imode*
+                        const Complex djt_cmplx = -2._rt * I*(i_new-1 + i + xmin*dxi)*wq*invdtdx/(amrex::Real)imode*
```

Comment:
to me more precise, this is mixing int and real operations and should explicitly cast to:
```C++
-2._rt * I*(amrex::Real(i_new-1 + i) + xmin*dxi)*wq*invdtdx/amrex::Real(imode)*
```